lib/prember-urls.js

```diff
@@ -0,0 +1,17 @@
+const jsdom = require("jsdom");
+const { JSDOM } = jsdom;
+
+module.exports = function urls(distDir, visit) {
+  return visit('/docs/introduction').then(page => {
+    return page.html();
+  }).then(html => {
+    var dom = new JSDOM(html);
+    var urls = ['/'];
+    for (let aTag of [...dom.window.document.querySelectorAll('.side-menu a')]) {
```

Comment:
This is where I'm deciding which links to follow when pre-rendering.

Reply:
(Anything we don't pre-render will still render normally, due to having a 404.html file. The only downside of that is that the status code is 404 instead of 200.)